arch/arm/boot/dts/ste-ux500-samsung-golden.dts

```diff
@@ -396,6 +422,25 @@
 		};
 	};
 
+	i2c-gpio-0 {
+		i2c_gpio_0_default: i2c_gpio_0 {
+			golden_cfg1 {
+				pins = "GPIO77",	/* TOUCHKEY_SCL */
+				       "GPIO78";	/* TOUCHKEY_SDA */
+				ste,config = <&gpio_in_nopull>;
+			};
+		};
+	};
+
+	tc360 {
+		tc360_default: tc360_default {
```

Comment:
I'd call this (and the node above this) `touchkey` instead of `tc360` to make it a bit more generic.

Reply:
I changed the name.